testing/s3minio/build.gradle.kts

```diff
@@ -41,11 +41,16 @@ dependencies {
 
   compileOnly(libs.errorprone.annotations)
 
-  testImplementation(libs.bundles.junit.testing)
-  testRuntimeOnly(libs.junit.jupiter.engine)
-  testImplementation(libs.hadoop.common) { withSparkExcludes() }
+  // TODO issue in JvmSuitePlugin, that these dependencies are _not_ inherited by default
```

Comment:
FYI I could not reproduce the problem. If I remove lines 45-47, I am still able to run all the integration tests in this project and in all dependent projects, e.g. nessie-gc-iceberg-files.

Reply:
yup - correct. probably a left-over